pkg/dockerutil/dockerutils.go

```diff
@@ -95,7 +107,8 @@ func NetworkExists(netName string) bool {
 func RemoveNetwork(netName string) error {
 	client := GetDockerClient()
 	networks, _ := client.ListNetworks()
-	var err error
+	// the loop below may not contain such a network
+	var err error = &docker.NoSuchNetwork{ID: netName}
```

Comment:
This is a fix for a bug that I introduced in #5508

When you run `ddev stop` or `ddev poweroff` several times in a row, `Network <name> removed` is shown each time, even if there is no such network.